integration-tests/pom.xml

```diff
@@ -208,6 +208,45 @@
     </profile>
 
     <profile>
+      <id>swiftTest</id>
+      <dependencies>
+        <dependency>
+          <groupId>org.tachyonproject</groupId>
+          <artifactId>tachyon-underfs-swift</artifactId>
+          <version>${project.version}</version>
+          <scope>test</scope>
+        </dependency>
+      </dependencies>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-surefire-plugin</artifactId>
+            <configuration>
+              <!--argLine>-XX:+HeapDumpOnOutOfMemoryError -XX:HeapDumpPath=/tmp/dump</argLine-->
+              <systemPropertyVariables>
+                <ufs>tachyon.underfs.swift.SwiftUnderStorageCluster</ufs>
+                <!-- Change "myAuthUrlKey" to point authentication URL -->
```

Comment:
line 229 `myAuthUrlKey` but in line 230 it is `myAuthUrl`, inconsistent.
also for `myAuthMethodKey` and `myContainerKey` (no such key)